docs/Release Notes.md

```diff
@@ -0,0 +1,17 @@
+# Release Notes
+Release notes are an important part of the release process and we aim to provide high quality notes with our software, by:
+1. _Writing the notes collaboratively_, because nobody knows how to describe a feature to the users better than the team that worked on it.
+2. _translating them_, so that more people are able to get them.
+
+
+## Release notes workflow
+1. In the root folder of every project, we maintain a `RELEASE-NOTES.txt` file that is updated by every PR that adds a relevant feature. As a reminder of developers and reviewers, we usually add a note about this in the project GitHub PR template. 
+2. The `get_prs_list_action` action in the `release-toolkit`  provides a list of all the merged PRs  in a range of tags. 
+3. During the code freeze, the release manager gets the release notes from `1` , uses `2` to get the list of merged PRs to verify/integrate the notes and write down the final copy.
+4. The copy is used to update the `metadata/release-notes.txt` file which contains the release notes for the current version.
```

Comment:
Would it be worth replacing "which contains the release notes for the current version" with "which contains _only_ the release notes for the current version"?

Might help with a little bit of clarity – right now the versions that have a bunch of past release notes _also_ technically contain the release notes for the current version 🤷‍♂

Reply:
👍